code/__DEFINES/preferences.dm

```diff
@@ -59,8 +59,9 @@
 #define PREFTOGGLE_2_SEE_ITEM_OUTLINES 	4096
 #define PREFTOGGLE_2_HIDE_ITEM_TOOLTIPS  8192
 #define PREFTOGGLE_2_THOUGHT_BUBBLE	16384
+#define PREFTOGGLE_2_MC_TABS	32768
```

Comment:
shouldn't these be converted to bit shifts? 
i.e
```dm
#define PREFTOGGLE_2_MC_TABS	(1 << 15)

Reply:
Yes